net/batman-adv-legacy/Makefile

```diff
@@ -19,7 +19,7 @@ BATCTL_MD5SUM:=42e269cc710bbc9a8fd17628201d4258
 PKG_SOURCE_PROTO:=git
 PKG_SOURCE_URL:=git://github.com/freifunk-gluon/batman-adv-legacy.git
 PKG_SOURCE_SUBDIR:=$(PKG_NAME)-$(PKG_VERSION)
-PKG_SOURCE_VERSION:=7b775e93b7d2d3f10b137e76090c82a06af65272
+PKG_SOURCE_VERSION:=0be9f10e49e3df113381c047d28ea5a5cfc1a0b6
```

Comment:
This commit ID needs to be updated, as the last PR was rebased.

Reply:
done